### DISCUSSION TITLE
“浅薄之患”的解法 | /posts/live-or-solve/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#596] [Announcements]
https://yuanfan.rbind.io/posts/live-or-solve/

**Comment by yihui** [comment]
今日整理浏览器里的文章理到了[黄永玉这篇](https://mp.weixin.qq.com/s/WtcjG3cuO93ugfI3gD8rWQ)，有两句话可以挑出来作为对狗主的回应：

> 不妨勇敢地活下去，不必求熟习世故哲学，不必八面玲珑来取得成功，毫无顾忌地接受挫折，不用作得失考虑，也不必作无效果的自救。

> 我与我周旋久，宁作我。

**Reply by earfanfan** [reply]
黄永玉那篇我有印象看过，印象特别深刻的是他有个朋友临行前给他打电话，他跟他朋友说叫他先不要死，就算要死也等他赶到了再死，结果是赶到的时候，朋友已经等不及先走了。

你挑出来这两句很有道理，要是十年前那个热血沸腾、无知无畏的我一定想都不想就觉得人生就应该是这样的，不过现在的我刚好进入了一段需要靠自己去摸索前进的旅程中，我想我还不能判断得到一种百分百的结论，比如某种自救一定是确定的无效果的所以不必作。

另外，我最近发现经历“浅薄之患”可能是一种必要的必经之路，因为现世真是个强者如林的世界啊，畏惧高山之远再正常不过了。貌似很久很久以前，大概是在一篇你写王国维三境界的古早博客，那时我在看山是山，两三年过去后，我也开始看山不是山了，约莫一晃眼再几十年过去后，我就可以跟活到九十多岁的黄永玉一样达到看山还是山的境界了吧，到那时候我应该也能坚定地宁作我。想到未来自己也许能活得那么豁达通透，还挺好的呢，不过现在还是继续活在现在吧。

**Reply by yihui** [reply]
就在刚刚！我把你前年那篇发在 COS 上的 echarts4r 教程转发给了一位医生。特来禀报一声。

**Reply by earfanfan** [reply]
哇嗷，大神又给我做宣传了，俺要夸下海口，猴年那个马月编一个超级大超级响的大马屁拍到大神的马腿上，哈哈。
